File: week1/fizz-buzz/fizzbuzz.js
function fizzBuzz() {
  let arr = []
  for(i = 1; i <= 100; i++) {
    if(i % 3 === 0 && i % 5 === 0) {
      arr.push("Fizzbuzz")
    } else if (i % 3 === 0) {
      arr.push("Fizz")
    } else if (i % 5 === 0) {
      arr.push("Buzz")
    } else {
      arr.push(i)
    }
  }
  return arr
}

console.log(fizzBuzz())

File: week1/fizz-buzz/fizzbuzz.py
def fizzbuzz():
    list = []
    for num in range(1, 101):
        if num % 3 == 0 and num % 5 == 0:
            list.append("Fizzbuzz")
        elif num % 5 == 0:
            list.append("Buzz")
        elif num % 3 == 0:
            list.append("Fizz")
        else:
            list.append(num)
    return list

